homeassistant/components/dynalite/bridge.py

```diff
@@ -61,6 +70,26 @@ def update_device(self, device: Optional[DynaliteBaseDevice] = None) -> None:
         else:
             async_dispatcher_send(self.hass, self.update_signal(device))
 
+    @callback
+    def handle_notification(self, notification: DynaliteNotification) -> None:
+        """Handle a notification from the platform and issue events."""
+        if notification.notification == NOTIFICATION_PACKET:
+            self.hass.bus.fire(
+                f"dynalite_{self.host}_packet",
+                {ATTR_PACKET: notification.data[NOTIFICATION_PACKET]},
+            )
+        if notification.notification == NOTIFICATION_PRESET:
+            self.hass.bus.fire(
+                f"dynalite_{self.host}_preset",
+                {
+                    ATTR_AREA: notification.data[dyn_CONF_AREA],
+                    ATTR_PRESET: notification.data[dyn_CONF_PRESET],
+                },
+            )
+            self.hass.bus.fire(
+                f"dynalite_{self.host}_selected_area_{notification.data[dyn_CONF_AREA]}_preset_{notification.data[dyn_CONF_PRESET]}"
```

Comment:
What's the point of this event without event data? Don't use variables in the event type.

Reply:
I wanted to make it easy to listen to a specific event. for example, if preset 4 in area 90 is turning the entire home off, i can listen to a specific string. it has the same information as the first event, just encoded in the event string. looking at your previous comments, probably better to only have the generic type with the data

Reply:
ok. deleted it and implemented the rest. please let me know if it works.

it has been a while since we communicated last. appreciate the consistent help